dspace-uclouvain/src/main/java/org/dspace/core/model/DialPerson.java

```diff
@@ -0,0 +1,109 @@
+package org.dspace.core.model;
+
+public class DialPerson {
+    public String alternatename = "";
+
+    public String firstname = "";
+    public String id_patron = "";
+    public String lastname = "";
+
+    public String orcid_id = "";
+
+    public DialPersonInscription inscription;
+
+    public String generateString(){
+        return this.alternatename + " " + this.firstname + " " + this.id_patron + " " + this.lastname;
+    }
+
+    public String getFullName(){
+        if(this.alternatename.isEmpty()){
+            return !this.firstname.isEmpty() ?  this.firstname + ", " + this.lastname: this.lastname;
+        }
+        return this.alternatename;
```

Comment:
not required to negate the `isEmpty()` function, just reverse ternary instructions.
not test empty alternateName, reverse and direct return
Cosmetic : for ternary, personnaly I like this (more readable) statement
```java
if (!this.alternateName.isEmpty()) {
    return this.alternateName;
}
return this.firstname.isEmpty()
    ? this.lastname
    : this.lastname + ", " + this.fistname;  //for DIAL, we use the format "lastname, firstname" ;-)
```